observability/cockpit/api-cli/configuring-grafana-agent.mdx

```diff
@@ -0,0 +1,178 @@
+---
+meta:
+  title: Configuring the Grafana agent
+  description: This page explains how to configure the Grafana agent
+content:
+  h1: Configuring the Grafana agent
+  paragraph: This page explains how to configure the Grafana agent
+tags: cockpit observability grafana-agent
+dates:
+  validation: 2023-01-05
+  posted: 2023-01-05
+categories:
+  - observability
+---
+
+This page explains how to configure the Grafana agent to push your logs and metrics and visualize them in Grafana.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the Scaleway (/console/my-account/how-to/create-an-account/)
+  - You have [configured your API keys](/identity-and-access-management/iam/how-to/create-api-keys/)
+  - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
+  - You have installed [docker](https://www.docker.com/) and [docker compose](https://docs.docker.com/compose/install/)
+</Message>
+
+## Configuring the Grafana agent
+
+You need your API secret key for the step below.
+
+1. Open up a terminal and run the following command to export your API key:
+
+    ```
+    export SCW_API_KEY="<your secret key>"
+    ```
+2. [Create a token](/observability/cockpit/how-to/create-token/) and select the push permission for metrics and logs.
+3. Create a folder where you will keep your configuration files.
+4. Create a configuration file inside your folder and name it `config.yaml`. This file will contain the Grafana agent configuration.
+5. Copy the following yaml template.
+    <Message type="important">
+     - Make sure you replace the `remote_write` URL with `https://metrics.prd.obs.fr-par.scw.cloud/api/v1/push` and the `clients` URL with `https://logs.prd.obs.fr-par.scw.cloud/loki/api/v1/push`
+     - Make sure you add the token you created in step 2
+    </Message>
+
+    ```
```

Comment:
Can you double check the identation of the snippet? looks pretty unlikely to me

Reply:
I was referring to the indentation of the yaml configuration file, sorry for the confusion.

Reply:
Thank you, I fixed it 👍🏼 <img width="769" alt="Capture d’écran 2023-01-09 à 14 35 04" src="https://user-images.githubusercontent.com/87707325/211320213-6f7b8909-6666-42ac-95c4-74828614242f.png">